src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -717,6 +717,16 @@ public enum NuGetLogCode
         /// </summary>
         NU5041 = 5041,
 
+        /// <summary>
+        /// Attempted to write files from multiple sources into the same location
+        /// </summary>
+        NU5042 = 5042,
```

Comment:
Where is this one used here?

Reply:
It's going to be NU5041. 

I hadn't rebased in a while, so I had to change the log code.